services/pkgreview/package-rating.service.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  name: Joi.string().required(),
+  type: Joi.string().required(),
+  icon: Joi.string().required(),
+  starString: [Joi.string().optional(), Joi.allow(null)],
+  reviewsCount: Joi.number(),
+  rating: [Joi.number().optional(), Joi.allow(null)],
+}).required()
+
+module.exports = class PkgreviewRating extends BaseJsonService {
+  static get category() {
+    return 'rating'
+  }
+
+  static get route() {
+    return {
+      base: 'pkgreview/rating',
+      pattern: ':pkgManager/:pkgSlug+',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'pkgreview.dev Package Ratings',
+        namedParams: { pkgManager: 'npm', pkgSlug: 'react' },
+        staticPreview: {
+          label: 'pkgreview.dev',
+          message: '4/5 (125)',
+          color: '#4F78FE',
+        },
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'pkgreview.dev',
+      color: '#4F78FE',
+    }
+  }
+
+  static render({ type, name, rating, reviewsCount }) {
+    return {
+      label: `pkgreview.dev`,
+      message: rating
+        ? `${+parseFloat(rating * 5).toFixed(2)}/5 (${reviewsCount})`
+        : 'Be the first to review',
```

Comment:
Please remember the information I shared on the issue around the need for these badges to be consistent with the way Shields handles such badges 😄 We do not inject additional text like this with links to a service/platform (see below)